rosbag2_cpp/test/rosbag2_cpp/test_info.cpp

```diff
@@ -273,3 +273,20 @@ TEST_F(TemporaryDirectoryFixture, read_metadata_makes_appropriate_call_to_metada
   EXPECT_EQ(read_metadata.compression_format, "zstd");
   EXPECT_EQ(read_metadata.compression_mode, "FILE");
 }
+
+TEST_F(TemporaryDirectoryFixture, info_for_standalone_bagfile) {
+  const auto bag_path = rcpputils::fs::path(temporary_dir_path_) / "bag";
+  const auto expected_bagfile_path = bag_path / "bag_0.db3";
+  {
+    // Create an empty bag with default storage
+    rosbag2_cpp::Writer writer;
+    writer.open(bag_path.string());
```

Comment:
Perhaps it would be better to specify sqlite3 storage for writer explicitly since we check for it at the end of the tests. 
In case if change default plugin to mcap this test is going to fail.

Reply:
I'd rather not specify explicitly, in the `mcap` switch I was planning to look through the whole codebase for the explicit string `sqlite3`, so we wouldn't miss this - but I do see the concern. The main thing I wanted to check was for some value that is not in `BagMetadata` structure default, to make sure that some storage thing happened. 

Two options:
* Maybe instead I should add a message and check for `message_count`?
* Add a `DEFAULT_STORAGE_PLUGIN` constant string somewhere that we can reference here.

Reply:
Adding topic and checking its name after open sounds good for me.
As regards the second option 
> Add a DEFAULT_STORAGE_PLUGIN constant string somewhere that we can reference here.

did you meant to add `DEFAULT_STORAGE_PLUGIN` variable in BagMetadata structure and fill it with the name of the storage plugin using during recording aka when we opening writer?

Reply:
No - I don't think that `BagMetadata` would be where that belongs. It might be appropriate as a compile definition, so that it is kept out of the code entirely. For now I will avoid this question and we can reconsider it later.